lib/audited/audit.rb

```diff
@@ -42,7 +42,7 @@ def text_column?
   class Audit < ::ActiveRecord::Base
     belongs_to :auditable, polymorphic: true
     belongs_to :user, polymorphic: true
-    belongs_to :associated, polymorphic: true
```

Comment:
This isn't referenced anywhere in this Gem's codebase? I guess it's probably more for ingesting in whatever it gets included in vs used internally.

Reply:
It was referenced a couple places before. This was how it got the associated records previously. Now it will use the `join` table

Reply:
Yep, makes sense.